saleor/graphql/account/types.py

```diff
@@ -379,3 +379,17 @@ class AddressValidationData(graphene.ObjectType):
     postal_code_matchers = graphene.List(graphene.String)
     postal_code_examples = graphene.List(graphene.String)
     postal_code_prefix = graphene.String()
+
+
+class StaffNotification(MetadataObjectType, CountableDjangoObjectType):
+    user = graphene.Field(User, description="Returns staff user.", required=False)
+    staff_email = graphene.String(description="Staff user's email.", required=False)
```

Comment:
Do we need two fields for email? Their descriptions are the same.

Reply:
`email` is property which takes `staff_email` or `use.email` depends on which field is set
I removed `staff_email` from type because it's not need to display this when `email` is present